backend/typescript/rest/camperRoutes.ts

```diff
@@ -66,4 +66,14 @@ camperRouter.get("/", async (req, res) => {
   }
 });
 
+/* Delete all campers with the chargeId with */
+camperRouter.put("/cancel/:chargeId", async (req, res) => {
```

Comment:
dkm can we change this to a delete endpoint, thinking about it again I think that makes more sense
```suggestion
camperRouter.delete("/cancel/:chargeId", async (req, res) => {
```